.github/ISSUE_TEMPLATE/new_python_tool_pip.yml

```
@@ -0,0 +1,128 @@
+name: 🆕 NEW PIP PACKAGE
+description: Propose the addition of a new Python package/tool that is installed with pip
+title: "pip package proposal: <package_name>.vm"
+labels: [":new: package"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for contributing to [VM-Packages](https://github.com/mandiant/VM-Packages), your package proposal supports tools available for [FLARE-VM](https://github.com/mandiant/flare-vm) and [CommandoVM](https://github.com/mandiant/commando-vm)! :cupid: Please ensure that your suggested tool doesn't already exist within the set of [current packages](https://github.com/mandiant/VM-Packages/tree/main/packages) and that there is no [issue](https://github.com/mandiant/VM-Packages/issues?q=is%3Aopen+is%3Aissue+label%3A%22%3Anew%3A+package%22) proposal already. If the tool is not related malware analysis, incident response, penetration testing and other security related tasking, consider using directly the [Chocolatey community package](https://community.chocolatey.org/packages) if there is one for the tool.
+  - type: input
+    id: pkg_name
+    attributes:
+      label: Package Name
+      description: |
+        The convention is to use lowercase names with the following format: `toolname` or `tool-name` and without `.vm` appended. Example: `js-deobfuscator`. **Please add this name to the issue title as well** (keep the `.vm` in the title).
+      placeholder: ex. js-deobfuscator
+    validations:
+      required: true
+  - type: input
+    id: tool_name
+    attributes:
+      label: Tool Name
+      description: |
+        The name of the tool being installed with `pip install <tool_name>`
+      placeholder: ex. magika
```

I think we should use the same tool in both `tool_name` and `pkg_name` to illustrate the difference between the two fields. Also, please add an example in the description as the placeholder is not shown after you start writing.
```suggestion
        The name of the tool being installed with `pip install <tool_name>`, Example: `autoit-ripper`.
      placeholder: ex. autoit-ripper
```